README.md

```diff
@@ -1 +1,203 @@
-# Capstone-1
\ No newline at end of file
+<h3><b>Portfolio</b></h3>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 Portfolio <a name="about-project"></a>
+
+Utilized CSS box model and Flexbox for layout design. Incorporated images and backgrounds for visual appeal. JavaScript-driven interactivity with DOM manipulation and events. Deployed using GitHub Pages. Demonstrates adaptability to various screen sizes with media queries.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">html5</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">CSS3</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">Javascript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Responsive Design**
+- **DOM Manipulatio**
+- **Animations and Transitions**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+ 
+- [Live Demo Link](https://abel12-pro.github.io/)
```

Comment:
- Kindly, you are using your Portfolio live demo link please deploy your website to GitHub pages and update the live demo link to make your readme professional 👍